### ISSUE TITLE
Game Crashes while Putting items into the Crafting Grid

### ISSUE THREAD

**Issue by alsk192837** [GTNewHorizons/GT-New-Horizons-Modpack#15332]
### Your Pack Version

2.5.1

### Your Server

SP

### Java Version

Java 8

### Type of Server

Single Player

### Your Actions

Game crashed when I put items into crafting grid(Any crafting grid like Vanilla Inventory, Vanilla Workbench, TiC Crafting Station etc.).
It seems like randomly, at the version 2.4.0, this bug existed and my world has been totally destroyed. The bug is still here.
The bug MAY be fixed when I shut down the game process (instead of quitting the world) and reöpen, but it did not fix itself on my last world.

### Crash Report

https://mclo.gs/vXtMKMp
I guess that game crashed because of the recipe cache.

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the crash still exists will prompt us to investigate and reopen it once we confirm your report.
- [ ] I can reproduce this crash consistently by following the exact steps I described above.
- [ ] I have asked other people and they confirm they also crash by following the exact steps I described above.

**Comment by chochem** [comment]
since it only seems to affect you so far: did you add or remove any mods, change recipe related configs, or added any recipe scripts?
also on the topic of scripts, is you script folder empty?

**Comment by alsk192837** [comment]
There is nothing in my scripts folder. I didn't change any scripts.
I changed my config folder by installing Chinese language pack.
I added Twist Space Technology, Manaita Plus and Deconstruction Table because mining ores is so fun.
I think it is about Forestry Crafting Table. I put items and game crashed AFTER finished using the forestry crafting table to make concrete bucket.
I find a mod called Gambiarra. Other players said it can fix this issue, but when I added this mod, game stopped loading on Witching Gadgets. I can't get into the game and check if it fixed.
However, the players who provided me this mod told me that the bug will only appeared on mixed server like Thermos, but I was playing on Singleplayer, so maybe the Internal Server crashed.
They also said the bug only happened on the 2x2 crafting slot, but the game crashed when I put items in not only 2x2 but also 3x3.

**Comment by alsk192837** [comment]
> since it only seems to affect you so far: did you add or remove any mods, change recipe related configs, or added any recipe scripts? also on the topic of scripts, is you script folder empty?

There is nothing in my scripts folder. I didn't change any scripts.
I changed my config folder by installing Chinese language pack.
I added Twist Space Technology, Manaita Plus and Deconstruction Table because mining ores is so fun.
I think it is about Forestry Crafting Table. I put items and game crashed AFTER finished using the forestry crafting table to make concrete bucket.
I find a mod called Gambiarra. Other players said it can fix this issue, but when I added this mod, game stopped loading on Witching Gadgets. I can't get into the game and check if it fixed.
However, the players who provided me this mod told me that the bug will only appeared on mixed server like Thermos, but I was playing on Singleplayer, so maybe the Internal Server crashed.
They also said the bug only happened on the 2x2 crafting slot, but the game crashed when I put items in not only 2x2 but also 3x3.
